packages/web/src/pages/upload-page/UploadPage.tsx

```diff
@@ -101,11 +101,13 @@ export const UploadPage = (props: UploadPageProps) => {
             formState={formState}
             onContinue={(formState: UploadFormState) => {
               setFormState(formState)
+              const isPrivateCollection =
```

Comment:
nit: i know `is_private` is only on collections and not tracks which is `is_unlisted`, but we could add an explicit check for `playlist_id` in here or something. i'm just imagining a future where we rename `is_unlisted` to `is_private` on tracks. up 2 u.